### ISSUE TITLE
shown size of drive is not correct

### ISSUE THREAD

**Issue by michiel1972** [jochemin/raspnode#14]
![image](https://user-images.githubusercontent.com/1163907/38052398-6dbb931e-32d1-11e8-98cd-fd0421dbd4e5.png)

It says 460M while it is 300Gb empty harddisk

**Comment by simonk83** [comment]
Mine also said 460M.  Led me down a bit of a rabbit hole until i just decided to ignore it :D

**Comment by jochemin** [comment]
Ok, it's because the hard drive is mounted. Thank you! working on it.

**Comment by vmavromatis** [comment]
Hey, I made https://github.com/jochemin/raspnode/pull/10 for this ;)

**Comment by jochemin** [comment]
Yes, vmavromatis, my concern is if the hard drive is not sda1 I don't know if it's possible to have a hard drive detected as sda2 or another way, in that case the script will fail.

**Comment by jochemin** [comment]
And I would like to handle if the hard drive has more than one partition.
